src/lib.rs

```diff
@@ -30,6 +30,17 @@
 //! these traits are `unsafe`, and you should carefully read the requirements
 //! before adding the them to your own types.
 //!
+//! ## Using Types from Extern Crates
```

Comment:
I don't think that we should document this idea.

It's possible, but it's kinda extra dodgy, particularly because it runs explicitly against the claimed rules of Pod and Zeroable.

Reply:
Yeah, I get your point. It's getting kinda convoluted.

But I don't know how obvious this use of `TransparentWrapper` is. It is a pretty nice feature to have, so people should know about it.

Maybe we need to reformulate the docs some more? I'll give it a try and if it looks bad, we probably have to remove it...

What is most desirable?
- Mentioning `TransparentWrapper` everywhere where it's relevant (so also in `Pod`).
- Being subtle about it and just mentioning it in it's own docs.
- Not mention it at all.

Reply:
I don't think we should mention it, since it's not sound unless you pin the version exactly. If other crates haven't opted into the bytemuck requirements, they're free to break them at any time without a semver-major change.

Reply:
Yeah, the reason that the docs for Pod and Zeroable require that the field type also is Pod or Zeroable is because then SemVer kicks in, specifically that it's a SemVer Break to remove a trait impl.

If you newtype a struct and declare the wrapper type Pod, then the inner type adds a `&'static str` field in a minor update (not unreasonable!) then suddenly you're hosed.

So I think that this should stay as a "secret thing the type system technically allows but officially you shouldn't so legally you can't complain when this blows up in your face (and it easily might)" sort of thing.

Reply:
Okay, I see. That's a reasonable argument.
In this case I'm going to remove all mentions of this.